### ISSUE TITLE
Failed tests - RHEL-6

### ISSUE THREAD

**Issue by remicollet** [llaville/php-compat-info#193]
Output from the generic test branch (which allow to report all issues in a single pass), with PHP -5.5.40 and a minimum set of extension installed

     Time: 51.31 seconds, Memory: 57.50Mb
     There were 36 failures:
     1) Bartlett\Tests\CompatInfo\Reference\Extension\CurlExtensionTest::testGetConstantsFromReference with data set #116 ('CURLINFO_LOCAL_IP', array('curl', '5.4.7', '', '5.4.7', '', array()))
     Constant 'CURLINFO_LOCAL_IP', found in Reference, does not exists.
     Failed asserting that false is true.
     /builddir/build/BUILD/php-compat-info-0ed258e3ed120704f40ec6461d5d61b6a5cd5971/tests/Reference/GenericTest.php:244
     /builddir/build/BUILD/php-compat-info-0ed258e3ed120704f40ec6461d5d61b6a5cd5971/tests/Reference/GenericTest.php:213
     /builddir/build/BUILD/php-compat-info-0ed258e3ed120704f40ec6461d5d61b6a5cd5971/tests/Reference/GenericTest.php:402
     2) Bartlett\Tests\CompatInfo\Reference\Extension\CurlExtensionTest::testGetConstantsFromReference with data set #117 ('CURLINFO_LOCAL_PORT', array('curl', '5.4.7', '', '5.4.7', '', array()))
     Constant 'CURLINFO_LOCAL_PORT', found in Reference, does not exists.
     Failed asserting that false is true.
     /builddir/build/BUILD/php-compat-info-0ed258e3ed120704f40ec6461d5d61b6a5cd5971/tests/Reference/GenericTest.php:244
     /builddir/build/BUILD/php-compat-info-0ed258e3ed120704f40ec6461d5d61b6a5cd5971/tests/Reference/GenericTest.php:213
     /builddir/build/BUILD/php-compat-info-0ed258e3ed120704f40ec6461d5d61b6a5cd5971/tests/Reference/GenericTest.php:402
     3) Bartlett\Tests\CompatInfo\Reference\Extension\CurlExtensionTest::testGetConstantsFromReference with data set #123 ('CURLINFO_PRIMARY_PORT', array('curl', '5.4.7', '', '5.4.7', '', array()))
     Constant 'CURLINFO_PRIMARY_PORT', found in Reference, does not exists.
     Failed asserting that false is true.
     /builddir/build/BUILD/php-compat-info-0ed258e3ed120704f40ec6461d5d61b6a5cd5971/tests/Reference/GenericTest.php:244
     /builddir/build/BUILD/php-compat-info-0ed258e3ed120704f40ec6461d5d61b6a5cd5971/tests/Reference/GenericTest.php:213
     /builddir/build/BUILD/php-compat-info-0ed258e3ed120704f40ec6461d5d61b6a5cd5971/tests/Reference/GenericTest.php:402
     4) Bartlett\Tests\CompatInfo\Reference\Extension\ZendopcacheExtensionTest::testGetIniEntriesFromReference with data set #0 ('opcache.enable', array('7.0.1', '', '5.2.0', ''))
     INI 'opcache.enable', found in Reference, does not exists.
     Failed asserting that false is not equal to false.
     /builddir/build/BUILD/php-compat-info-0ed258e3ed120704f40ec6461d5d61b6a5cd5971/tests/Reference/GenericTest.php:232
     /builddir/build/BUILD/php-compat-info-0ed258e3ed120704f40ec6461d5d61b6a5cd5971/tests/Reference/GenericTest.php:213
     /builddir/build/BUILD/php-compat-info-0ed258e3ed120704f40ec6461d5d61b6a5cd5971/tests/Reference/GenericTest.php:309
     5) Bartlett\Tests\CompatInfo\Reference\Extension\ZendopcacheExtensionTest::testGetIniEntriesFromReference with data set #1 ('opcache.use_cwd', array('7.0.1', '', '5.2.0', ''))
     INI 'opcache.use_cwd', found in Reference, does not exists.
     Failed asserting that false is not equal to false.
     /builddir/build/BUILD/php-compat-info-0ed258e3ed120704f40ec6461d5d61b6a5cd5971/tests/Reference/GenericTest.php:232
     /builddir/build/BUILD/php-compat-info-0ed258e3ed120704f40ec6461d5d61b6a5cd5971/tests/Reference/GenericTest.php:213
     /builddir/build/BUILD/php-compat-info-0ed258e3ed120704f40ec6461d5d61b6a5cd5971/tests/Reference/GenericTest.php:309
     6) Bartlett\Tests\CompatInfo\Reference\Extension\ZendopcacheExtensionTest::testGetIniEntriesFromReference with data set #2 ('opcache.validate_timestamps', array('7.0.1', '', '5.2.0', ''))
     INI 'opcache.validate_timestamps', found in Reference, does not exists.
     Failed asserting that false is not equal to false.
     /builddir/build/BUILD/php-compat-info-0ed258e3ed120704f40ec6461d5d61b6a5cd5971/tests/Reference/GenericTest.php:232
     /builddir/build/BUILD/php-compat-info-0ed258e3ed120704f40ec6461d5d61b6a5cd5971/tests/Reference/GenericTest.php:213
     /builddir/build/BUILD/php-compat-info-0ed258e3ed120704f40ec6461d5d61b6a5cd5971/tests/Reference/GenericTest.php:309
     7) Bartlett\Tests\CompatInfo\Reference\Extension\ZendopcacheExtensionTest::testGetIniEntriesFromReference with data set #3 ('opcache.inherited_hack', array('7.0.1', '', '5.2.0', ''))
     INI 'opcache.inherited_hack', found in Reference, does not exists.
     Failed asserting that false is not equal to false.
     /builddir/build/BUILD/php-compat-info-0ed258e3ed120704f40ec6461d5d61b6a5cd5971/tests/Reference/GenericTest.php:232
     /builddir/build/BUILD/php-compat-info-0ed258e3ed120704f40ec6461d5d61b6a5cd5971/tests/Reference/GenericTest.php:213
     /builddir/build/BUILD/php-compat-info-0ed258e3ed120704f40ec6461d5d61b6a5cd5971/tests/Reference/GenericTest.php:309
     8) Bartlett\Tests\CompatInfo\Reference\Extension\ZendopcacheExtensionTest::testGetIniEntriesFromReference with data set #4 ('opcache.dups_fix', array('7.0.1', '', '5.2.0', ''))
     INI 'opcache.dups_fix', found in Reference, does not exists.
     Failed asserting that false is not equal to false.
     /builddir/build/BUILD/php-compat-info-0ed258e3ed120704f40ec6461d5d61b6a5cd5971/tests/Reference/GenericTest.php:232
     /builddir/build/BUILD/php-compat-info-0ed258e3ed120704f40ec6461d5d61b6a5cd5971/tests/Reference/GenericTest.php:213
     /builddir/build/BUILD/php-compat-info-0ed258e3ed120704f40ec6461d5d61b6a5cd5971/tests/Reference/GenericTest.php:309
     9) Bartlett\Tests\CompatInfo\Reference\Extension\ZendopcacheExtensionTest::testGetIniEntriesFromReference with data set #5 ('opcache.revalidate_path', array('7.0.1', '', '5.2.0', ''))
     INI 'opcache.revalidate_path', found in Reference, does not exists.
     Failed asserting that false is not equal to false.
     /builddir/build/BUILD/php-compat-info-0ed258e3ed120704f40ec6461d5d61b6a5cd5971/tests/Reference/GenericTest.php:232
     /builddir/build/BUILD/php-compat-info-0ed258e3ed120704f40ec6461d5d61b6a5cd5971/tests/Reference/GenericTest.php:213
     /builddir/build/BUILD/php-compat-info-0ed258e3ed120704f40ec6461d5d61b6a5cd5971/tests/Reference/GenericTest.php:309
     10) Bartlett\Tests\CompatInfo\Reference\Extension\ZendopcacheExtensionTest::testGetIniEntriesFromReference with data set #6 ('opcache.log_verbosity_level', array('7.0.1', '', '5.2.0', ''))
     INI 'opcache.log_verbosity_level', found in Reference, does not exists.
     Failed asserting that false is not equal to false.
     /builddir/build/BUILD/php-compat-info-0ed258e3ed120704f40ec6461d5d61b6a5cd5971/tests/Reference/GenericTest.php:232
     /builddir/build/BUILD/php-compat-info-0ed258e3ed120704f40ec6461d5d61b6a5cd5971/tests/Reference/GenericTest.php:213
     /builddir/build/BUILD/php-compat-info-0ed258e3ed120704f40ec6461d5d61b6a5cd5971/tests/Reference/GenericTest.php:309
     11) Bartlett\Tests\CompatInfo\Reference\Extension\ZendopcacheExtensionTest::testGetIniEntriesFromReference with data set #7 ('opcache.memory_consumption', array('7.0.1', '', '5.2.0', ''))
     INI 'opcache.memory_consumption', found in Reference, does not exists.
     Failed asserting that false is not equal to false.
     /builddir/build/BUILD/php-compat-info-0ed258e3ed120704f40ec6461d5d61b6a5cd5971/tests/Reference/GenericTest.php:232
     /builddir/build/BUILD/php-compat-info-0ed258e3ed120704f40ec6461d5d61b6a5cd5971/tests/Reference/GenericTest.php:213
     /builddir/build/BUILD/php-compat-info-0ed258e3ed120704f40ec6461d5d61b6a5cd5971/tests/Reference/GenericTest.php:309
     12) Bartlett\Tests\CompatInfo\Reference\Extension\ZendopcacheExtensionTest::testGetIniEntriesFromReference with data set #8 ('opcache.interned_strings_buffer', array('7.0.1', '', '5.2.0', ''))
     INI 'opcache.interned_strings_buffer', found in Reference, does not exists.
     Failed asserting that false is not equal to false.
     /builddir/build/BUILD/php-compat-info-0ed258e3ed120704f40ec6461d5d61b6a5cd5971/tests/Reference/GenericTest.php:232
     /builddir/build/BUILD/php-compat-info-0ed258e3ed120704f40ec6461d5d61b6a5cd5971/tests/Reference/GenericTest.php:213
     /builddir/build/BUILD/php-compat-info-0ed258e3ed120704f40ec6461d5d61b6a5cd5971/tests/Reference/GenericTest.php:309
     13) Bartlett\Tests\CompatInfo\Reference\Extension\ZendopcacheExtensionTest::testGetIniEntriesFromReference with data set #9 ('opcache.max_accelerated_files', array('7.0.1', '', '5.2.0', ''))
     INI 'opcache.max_accelerated_files', found in Reference, does not exists.
     Failed asserting that false is not equal to false.
     /builddir/build/BUILD/php-compat-info-0ed258e3ed120704f40ec6461d5d61b6a5cd5971/tests/Reference/GenericTest.php:232
     /builddir/build/BUILD/php-compat-info-0ed258e3ed120704f40ec6461d5d61b6a5cd5971/tests/Reference/GenericTest.php:213
     /builddir/build/BUILD/php-compat-info-0ed258e3ed120704f40ec6461d5d61b6a5cd5971/tests/Reference/GenericTest.php:309
     14) Bartlett\Tests\CompatInfo\Reference\Extension\ZendopcacheExtensionTest::testGetIniEntriesFromReference with data set #10 ('opcache.max_wasted_percentage', array('7.0.1', '', '5.2.0', ''))
     INI 'opcache.max_wasted_percentage', found in Reference, does not exists.
     Failed asserting that false is not equal to false.
     /builddir/build/BUILD/php-compat-info-0ed258e3ed120704f40ec6461d5d61b6a5cd5971/tests/Reference/GenericTest.php:232
     /builddir/build/BUILD/php-compat-info-0ed258e3ed120704f40ec6461d5d61b6a5cd5971/tests/Reference/GenericTest.php:213
     /builddir/build/BUILD/php-compat-info-0ed258e3ed120704f40ec6461d5d61b6a5cd5971/tests/Reference/GenericTest.php:309
     15) Bartlett\Tests\CompatInfo\Reference\Extension\ZendopcacheExtensionTest::testGetIniEntriesFromReference with data set #11 ('opcache.consistency_checks', array('7.0.1', '', '5.2.0', ''))
     INI 'opcache.consistency_checks', found in Reference, does not exists.
     Failed asserting that false is not equal to false.
     /builddir/build/BUILD/php-compat-info-0ed258e3ed120704f40ec6461d5d61b6a5cd5971/tests/Reference/GenericTest.php:232
     /builddir/build/BUILD/php-compat-info-0ed258e3ed120704f40ec6461d5d61b6a5cd5971/tests/Reference/GenericTest.php:213
     /builddir/build/BUILD/php-compat-info-0ed258e3ed120704f40ec6461d5d61b6a5cd5971/tests/Reference/GenericTest.php:309
     16) Bartlett\Tests\CompatInfo\Reference\Extension\ZendopcacheExtensionTest::testGetIniEntriesFromReference with data set #12 ('opcache.force_restart_timeout', array('7.0.1', '', '5.2.0', ''))
     INI 'opcache.force_restart_timeout', found in Reference, does not exists.
     Failed asserting that false is not equal to false.
     /builddir/build/BUILD/php-compat-info-0ed258e3ed120704f40ec6461d5d61b6a5cd5971/tests/Reference/GenericTest.php:232
     /builddir/build/BUILD/php-compat-info-0ed258e3ed120704f40ec6461d5d61b6a5cd5971/tests/Reference/GenericTest.php:213
     /builddir/build/BUILD/php-compat-info-0ed258e3ed120704f40ec6461d5d61b6a5cd5971/tests/Reference/GenericTest.php:309
     17) Bartlett\Tests\CompatInfo\Reference\Extension\ZendopcacheExtensionTest::testGetIniEntriesFromReference with data set #13 ('opcache.revalidate_freq', array('7.0.1', '', '5.2.0', ''))
     INI 'opcache.revalidate_freq', found in Reference, does not exists.
     Failed asserting that false is not equal to false.
     /builddir/build/BUILD/php-compat-info-0ed258e3ed120704f40ec6461d5d61b6a5cd5971/tests/Reference/GenericTest.php:232
     /builddir/build/BUILD/php-compat-info-0ed258e3ed120704f40ec6461d5d61b6a5cd5971/tests/Reference/GenericTest.php:213
     /builddir/build/BUILD/php-compat-info-0ed258e3ed120704f40ec6461d5d61b6a5cd5971/tests/Reference/GenericTest.php:309
     18) Bartlett\Tests\CompatInfo\Reference\Extension\ZendopcacheExtensionTest::testGetIniEntriesFromReference with data set #14 ('opcache.preferred_memory_model', array('7.0.1', '', '5.2.0', ''))
     INI 'opcache.preferred_memory_model', found in Reference, does not exists.
     Failed asserting that false is not equal to false.
     /builddir/build/BUILD/php-compat-info-0ed258e3ed120704f40ec6461d5d61b6a5cd5971/tests/Reference/GenericTest.php:232
     /builddir/build/BUILD/php-compat-info-0ed258e3ed120704f40ec6461d5d61b6a5cd5971/tests/Reference/GenericTest.php:213
     /builddir/build/BUILD/php-compat-info-0ed258e3ed120704f40ec6461d5d61b6a5cd5971/tests/Reference/GenericTest.php:309
     19) Bartlett\Tests\CompatInfo\Reference\Extension\ZendopcacheExtensionTest::testGetIniEntriesFromReference with data set #15 ('opcache.blacklist_filename', array('7.0.1', '', '5.2.0', ''))
     INI 'opcache.blacklist_filename', found in Reference, does not exists.
     Failed asserting that false is not equal to false.
     /builddir/build/BUILD/php-compat-info-0ed258e3ed120704f40ec6461d5d61b6a5cd5971/tests/Reference/GenericTest.php:232
     /builddir/build/BUILD/php-compat-info-0ed258e3ed120704f40ec6461d5d61b6a5cd5971/tests/Reference/GenericTest.php:213
     /builddir/build/BUILD/php-compat-info-0ed258e3ed120704f40ec6461d5d61b6a5cd5971/tests/Reference/GenericTest.php:309
     20) Bartlett\Tests\CompatInfo\Reference\Extension\ZendopcacheExtensionTest::testGetIniEntriesFromReference with data set #16 ('opcache.max_file_size', array('7.0.1', '', '5.2.0', ''))
     INI 'opcache.max_file_size', found in Reference, does not exists.
     Failed asserting that false is not equal to false.
     /builddir/build/BUILD/php-compat-info-0ed258e3ed120704f40ec6461d5d61b6a5cd5971/tests/Reference/GenericTest.php:232
     /builddir/build/BUILD/php-compat-info-0ed258e3ed120704f40ec6461d5d61b6a5cd5971/tests/Reference/GenericTest.php:213
     /builddir/build/BUILD/php-compat-info-0ed258e3ed120704f40ec6461d5d61b6a5cd5971/tests/Reference/GenericTest.php:309
     21) Bartlett\Tests\CompatInfo\Reference\Extension\ZendopcacheExtensionTest::testGetIniEntriesFromReference with data set #17 ('opcache.protect_memory', array('7.0.1', '', '5.2.0', ''))
     INI 'opcache.protect_memory', found in Reference, does not exists.
     Failed asserting that false is not equal to false.
     /builddir/build/BUILD/php-compat-info-0ed258e3ed120704f40ec6461d5d61b6a5cd5971/tests/Reference/GenericTest.php:232
     /builddir/build/BUILD/php-compat-info-0ed258e3ed120704f40ec6461d5d61b6a5cd5971/tests/Reference/GenericTest.php:213
     /builddir/build/BUILD/php-compat-info-0ed258e3ed120704f40ec6461d5d61b6a5cd5971/tests/Reference/GenericTest.php:309
     22) Bartlett\Tests\CompatInfo\Reference\Extension\ZendopcacheExtensionTest::testGetIniEntriesFromReference with data set #18 ('opcache.save_comments', array('7.0.1', '', '5.2.0', ''))
     INI 'opcache.save_comments', found in Reference, does not exists.
     Failed asserting that false is not equal to false.
     /builddir/build/BUILD/php-compat-info-0ed258e3ed120704f40ec6461d5d61b6a5cd5971/tests/Reference/GenericTest.php:232
     /builddir/build/BUILD/php-compat-info-0ed258e3ed120704f40ec6461d5d61b6a5cd5971/tests/Reference/GenericTest.php:213
     /builddir/build/BUILD/php-compat-info-0ed258e3ed120704f40ec6461d5d61b6a5cd5971/tests/Reference/GenericTest.php:309
     23) Bartlett\Tests\CompatInfo\Reference\Extension\ZendopcacheExtensionTest::testGetIniEntriesFromReference with data set #19 ('opcache.load_comments', array('7.0.1', '', '5.2.0', ''))
     INI 'opcache.load_comments', found in Reference, does not exists.
     Failed asserting that false is not equal to false.
     /builddir/build/BUILD/php-compat-info-0ed258e3ed120704f40ec6461d5d61b6a5cd5971/tests/Reference/GenericTest.php:232
     /builddir/build/BUILD/php-compat-info-0ed258e3ed120704f40ec6461d5d61b6a5cd5971/tests/Reference/GenericTest.php:213
     /builddir/build/BUILD/php-compat-info-0ed258e3ed120704f40ec6461d5d61b6a5cd5971/tests/Reference/GenericTest.php:309
     24) Bartlett\Tests\CompatInfo\Reference\Extension\ZendopcacheExtensionTest::testGetIniEntriesFromReference with data set #20 ('opcache.fast_shutdown', array('7.0.1', '', '5.2.0', ''))
     INI 'opcache.fast_shutdown', found in Reference, does not exists.
     Failed asserting that false is not equal to false.
     /builddir/build/BUILD/php-compat-info-0ed258e3ed120704f40ec6461d5d61b6a5cd5971/tests/Reference/GenericTest.php:232
     /builddir/build/BUILD/php-compat-info-0ed258e3ed120704f40ec6461d5d61b6a5cd5971/tests/Reference/GenericTest.php:213
     /builddir/build/BUILD/php-compat-info-0ed258e3ed120704f40ec6461d5d61b6a5cd5971/tests/Reference/GenericTest.php:309
     25) Bartlett\Tests\CompatInfo\Reference\Extension\ZendopcacheExtensionTest::testGetIniEntriesFromReference with data set #21 ('opcache.optimization_level', array('7.0.1', '', '5.2.0', ''))
     INI 'opcache.optimization_level', found in Reference, does not exists.
     Failed asserting that false is not equal to false.
     /builddir/build/BUILD/php-compat-info-0ed258e3ed120704f40ec6461d5d61b6a5cd5971/tests/Reference/GenericTest.php:232
     /builddir/build/BUILD/php-compat-info-0ed258e3ed120704f40ec6461d5d61b6a5cd5971/tests/Reference/GenericTest.php:213
     /builddir/build/BUILD/php-compat-info-0ed258e3ed120704f40ec6461d5d61b6a5cd5971/tests/Reference/GenericTest.php:309
     26) Bartlett\Tests\CompatInfo\Reference\Extension\ZendopcacheExtensionTest::testGetIniEntriesFromReference with data set #22 ('opcache.enable_file_override', array('7.0.1', '', '5.2.0', ''))
     INI 'opcache.enable_file_override', found in Reference, does not exists.
     Failed asserting that false is not equal to false.
     /builddir/build/BUILD/php-compat-info-0ed258e3ed120704f40ec6461d5d61b6a5cd5971/tests/Reference/GenericTest.php:232
     /builddir/build/BUILD/php-compat-info-0ed258e3ed120704f40ec6461d5d61b6a5cd5971/tests/Reference/GenericTest.php:213
     /builddir/build/BUILD/php-compat-info-0ed258e3ed120704f40ec6461d5d61b6a5cd5971/tests/Reference/GenericTest.php:309
     27) Bartlett\Tests\CompatInfo\Reference\Extension\ZendopcacheExtensionTest::testGetIniEntriesFromReference with data set #23 ('opcache.enable_cli', array('7.0.1', '', '5.2.0', ''))
     INI 'opcache.enable_cli', found in Reference, does not exists.
     Failed asserting that false is not equal to false.
     /builddir/build/BUILD/php-compat-info-0ed258e3ed120704f40ec6461d5d61b6a5cd5971/tests/Reference/GenericTest.php:232
     /builddir/build/BUILD/php-compat-info-0ed258e3ed120704f40ec6461d5d61b6a5cd5971/tests/Reference/GenericTest.php:213
     /builddir/build/BUILD/php-compat-info-0ed258e3ed120704f40ec6461d5d61b6a5cd5971/tests/Reference/GenericTest.php:309
     28) Bartlett\Tests\CompatInfo\Reference\Extension\ZendopcacheExtensionTest::testGetIniEntriesFromReference with data set #24 ('opcache.error_log', array('7.0.1', '', '5.2.0', ''))
     INI 'opcache.error_log', found in Reference, does not exists.
     Failed asserting that false is not equal to false.
     /builddir/build/BUILD/php-compat-info-0ed258e3ed120704f40ec6461d5d61b6a5cd5971/tests/Reference/GenericTest.php:232
     /builddir/build/BUILD/php-compat-info-0ed258e3ed120704f40ec6461d5d61b6a5cd5971/tests/Reference/GenericTest.php:213
     /builddir/build/BUILD/php-compat-info-0ed258e3ed120704f40ec6461d5d61b6a5cd5971/tests/Reference/GenericTest.php:309
     29) Bartlett\Tests\CompatInfo\Reference\Extension\ZendopcacheExtensionTest::testGetIniEntriesFromReference with data set #26 ('opcache.file_update_protection', array('7.0.3', '', '5.2.0', ''))
     INI 'opcache.file_update_protection', found in Reference, does not exists.
     Failed asserting that false is not equal to false.
     /builddir/build/BUILD/php-compat-info-0ed258e3ed120704f40ec6461d5d61b6a5cd5971/tests/Reference/GenericTest.php:232
     /builddir/build/BUILD/php-compat-info-0ed258e3ed120704f40ec6461d5d61b6a5cd5971/tests/Reference/GenericTest.php:213
     /builddir/build/BUILD/php-compat-info-0ed258e3ed120704f40ec6461d5d61b6a5cd5971/tests/Reference/GenericTest.php:309
     30) Bartlett\Tests\CompatInfo\Reference\Extension\ZendopcacheExtensionTest::testGetIniEntriesFromReference with data set #27 ('opcache.restrict_api', array('7.0.3', '', '5.2.0', ''))
     INI 'opcache.restrict_api', found in Reference, does not exists.
     Failed asserting that false is not equal to false.
     /builddir/build/BUILD/php-compat-info-0ed258e3ed120704f40ec6461d5d61b6a5cd5971/tests/Reference/GenericTest.php:232
     /builddir/build/BUILD/php-compat-info-0ed258e3ed120704f40ec6461d5d61b6a5cd5971/tests/Reference/GenericTest.php:213
     /builddir/build/BUILD/php-compat-info-0ed258e3ed120704f40ec6461d5d61b6a5cd5971/tests/Reference/GenericTest.php:309
     31) Bartlett\Tests\CompatInfo\Reference\Extension\ZendopcacheExtensionTest::testGetFunctionsFromReference with data set #0 ('opcache_reset', array('Zend OPcache', '7.0.1', '', '5.2.0', '', '', array()))
     Function 'opcache_reset', found in Reference, does not exists.
     Failed asserting that false is true.
     /builddir/build/BUILD/php-compat-info-0ed258e3ed120704f40ec6461d5d61b6a5cd5971/tests/Reference/GenericTest.php:238
     /builddir/build/BUILD/php-compat-info-0ed258e3ed120704f40ec6461d5d61b6a5cd5971/tests/Reference/GenericTest.php:213
     /builddir/build/BUILD/php-compat-info-0ed258e3ed120704f40ec6461d5d61b6a5cd5971/tests/Reference/GenericTest.php:357
     32) Bartlett\Tests\CompatInfo\Reference\Extension\ZendopcacheExtensionTest::testGetFunctionsFromReference with data set #1 ('opcache_invalidate', array('Zend OPcache', '7.0.2', '', '5.2.0', '', '', array()))
     Function 'opcache_invalidate', found in Reference, does not exists.
     Failed asserting that false is true.
     /builddir/build/BUILD/php-compat-info-0ed258e3ed120704f40ec6461d5d61b6a5cd5971/tests/Reference/GenericTest.php:238
     /builddir/build/BUILD/php-compat-info-0ed258e3ed120704f40ec6461d5d61b6a5cd5971/tests/Reference/GenericTest.php:213
     /builddir/build/BUILD/php-compat-info-0ed258e3ed120704f40ec6461d5d61b6a5cd5971/tests/Reference/GenericTest.php:357
     33) Bartlett\Tests\CompatInfo\Reference\Extension\ZendopcacheExtensionTest::testGetFunctionsFromReference with data set #2 ('opcache_compile_file', array('Zend OPcache', '7.0.3', '', '5.2.0', '', '', array()))
     Function 'opcache_compile_file', found in Reference, does not exists.
     Failed asserting that false is true.
     /builddir/build/BUILD/php-compat-info-0ed258e3ed120704f40ec6461d5d61b6a5cd5971/tests/Reference/GenericTest.php:238
     /builddir/build/BUILD/php-compat-info-0ed258e3ed120704f40ec6461d5d61b6a5cd5971/tests/Reference/GenericTest.php:213
     /builddir/build/BUILD/php-compat-info-0ed258e3ed120704f40ec6461d5d61b6a5cd5971/tests/Reference/GenericTest.php:357
     34) Bartlett\Tests\CompatInfo\Reference\Extension\ZendopcacheExtensionTest::testGetFunctionsFromReference with data set #3 ('opcache_is_script_cached', array('Zend OPcache', '7.0.4', '', '5.2.0', '', '', array()))
     Function 'opcache_is_script_cached', found in Reference, does not exists.
     Failed asserting that false is true.
     /builddir/build/BUILD/php-compat-info-0ed258e3ed120704f40ec6461d5d61b6a5cd5971/tests/Reference/GenericTest.php:238
     /builddir/build/BUILD/php-compat-info-0ed258e3ed120704f40ec6461d5d61b6a5cd5971/tests/Reference/GenericTest.php:213
     /builddir/build/BUILD/php-compat-info-0ed258e3ed120704f40ec6461d5d61b6a5cd5971/tests/Reference/GenericTest.php:357
     35) Bartlett\Tests\CompatInfo\Reference\Extension\ZendopcacheExtensionTest::testGetFunctionsFromReference with data set #4 ('opcache_get_configuration', array('Zend OPcache', '7.0.1', '', '5.2.0', '', '', array()))
     Function 'opcache_get_configuration', found in Reference, does not exists.
     Failed asserting that false is true.
     /builddir/build/BUILD/php-compat-info-0ed258e3ed120704f40ec6461d5d61b6a5cd5971/tests/Reference/GenericTest.php:238
     /builddir/build/BUILD/php-compat-info-0ed258e3ed120704f40ec6461d5d61b6a5cd5971/tests/Reference/GenericTest.php:213
     /builddir/build/BUILD/php-compat-info-0ed258e3ed120704f40ec6461d5d61b6a5cd5971/tests/Reference/GenericTest.php:357
     36) Bartlett\Tests\CompatInfo\Reference\Extension\ZendopcacheExtensionTest::testGetFunctionsFromReference with data set #5 ('opcache_get_status', array('Zend OPcache', '7.0.1', '', '5.2.0', '', '', array()))
     Function 'opcache_get_status', found in Reference, does not exists.
     Failed asserting that false is true.
     /builddir/build/BUILD/php-compat-info-0ed258e3ed120704f40ec6461d5d61b6a5cd5971/tests/Reference/GenericTest.php:238
     /builddir/build/BUILD/php-compat-info-0ed258e3ed120704f40ec6461d5d61b6a5cd5971/tests/Reference/GenericTest.php:213
     /builddir/build/BUILD/php-compat-info-0ed258e3ed120704f40ec6461d5d61b6a5cd5971/tests/Reference/GenericTest.php:357
     FAILURES!
     Tests: 644192, Assertions: 4050, Failures: 36, Skipped: 4156.

**Comment by remicollet** [comment]
* as #191, opcache issues
* optional constant with curl 7.19

**Comment by llaville** [comment]
Results from my windows laptop
```
Time: 8.08 minutes, Memory: 92.25Mb, References: 105

Results KO. Tests: 7018, Assertions: 8230, Failures: 59, Skipped: 316
```
With all events logged to  `phpunit-phpcompatinfo-php50440-20150505.log` : this is now my base milestone to check optional elements and all others specificities

**Comment by llaville** [comment]
This results are certainly now outdated, and I get no feedback, so it won't be included in final version 4.2.0
